lib/govuk_publishing_components/app_helpers/asset_helper.rb

```diff
@@ -0,0 +1,155 @@
+module GovukPublishingComponents
+  module AppHelpers
+    class AssetHelper
```

Comment:
We should have some unit tests for this class

Reply:
Done 👍